src/main/java/org/folio/rest/impl/OaiPmhViewInstancesAPI.java

```diff
@@ -80,6 +96,30 @@ public void getOaiPmhViewInstances(String startDate, String endDate, boolean del
     }
   }
 
+  @Validate
+  @Override
+  public void postOaiPmhViewEnrichedInstances(OaipmhInstanceIds entity, RoutingContext routingContext,
+      Map<String, String> okapiHeaders, Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
+
+    UUID[] ids = entity.getInstanceids().stream().map(UUID::fromString).toArray(UUID[]::new);
```

Comment:
Maybe directly use `String[]` here? assuming that the payload might be quite big.

Reply:
Usaing of String array allows us to eliminate extra conversion, because later array if UUIDs will be passed as array of Strings in payload. But internally UUID type is used to be sure that passed data has UUID type, ans not just any strings. Possibly this is not a big overhead, I mean conversion from String to UUID and then back to String for payload, because data consistency is on the other side. And the by fact, we deal with UUID therefore I believe we need to use this type of data inside the code.

Reply:
Ok, let's leave as it is.

Reply:
What happens if a client provides a ID that is not a valid UUID?